authentication/src/main/AndroidManifest.xml

```diff
@@ -1,18 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.udinic.accounts_authenticator_example"
+          package="com.udinic.accounts"
```

Comment:
"accounts" is too generic. I see myself using it for a real app.
How about "auth_example" ?